src/lang/std/artifactGraph.ts

```diff
@@ -587,13 +590,15 @@ export function expandPath(
     { keys: path.segIds, types: ['segment'] },
     artifactGraph
   )
-  const sweep = getArtifactOfTypes(
-    {
-      key: path.sweepId,
-      types: ['sweep'],
-    },
-    artifactGraph
-  )
+  const sweep = path.sweepId
+    ? getArtifactOfTypes(
+        {
+          key: path.sweepId,
+          types: ['sweep'],
+        },
+        artifactGraph
+      )
+    : undefined
```

Comment:
Won't this always be truthy because of the function signature?

```ts
export function getArtifactOfTypes<T extends Artifact['type'][]>(
  {
    key,
    types,
  }: {
    key: ArtifactId
    types: T
  },
  map: ArtifactGraph
): Extract<Artifact, { type: T[number] }> | Error
```

Even an `Error` will be truthy

Reply:
`path.sweepId` will either be a `string` or `undefined` though

Reply:
🤦 my bad, I read this wrong!